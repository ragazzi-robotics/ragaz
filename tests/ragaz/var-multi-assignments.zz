def main():
    var a, b, c, d
    var e: int, f: int = c, d = a, b = 1, 2.5
    print(a)
    print(b)
    print(c)
    print(d)
    print(e)
    print(f)
