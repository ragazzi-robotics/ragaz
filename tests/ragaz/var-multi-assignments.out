1
2.500000
1
2.500000
1
2
